main/reference/zoe-api/zoe-contract-facet.md

```diff
@@ -100,25 +106,18 @@ The optional **customDetails** argument is included in the **Invitation**'s
 **amount** and not otherwise relied on by Zoe.
 
 The optional **proposalShape** argument can be used to describe the required and allowed components of each proposal.
+
+### Proposal Shapes
+
 Proposals that don't match the pattern will be rejected by Zoe without even being sent to the contract.
 
```

Comment:
```suggestion
The optional **proposalShape** argument can be used to describe the required and allowed components of each proposal.
Proposals that don't match the provided pattern will be rejected by Zoe without even being sent to the contract.

### Proposal Shapes

```

Reply:
I think it was like that before; doesn't that mean people have to read _before_ the heading to get the main point?